app/views/shared/_alert.erb

```diff
@@ -0,0 +1,6 @@
+<% flash.each do |key, message| %>
+<div class="metrics-alert alert alert-warning alert-dismissible fade show" role="alert">
```

Comment:
Why `alert-warning` is hardcoded? the flash can be a warning, a notice, an error, etc, right?

Reply:
In this app only need one warning.